facilitator/src/bin/facilitator.rs

```diff
@@ -2,7 +2,7 @@ use anyhow::{anyhow, Context, Result};
 use chrono::{prelude::Utc, NaiveDateTime};
```

Comment:
These changes to `facilitator` make sense, but they will break existing paired test envs until the corresponding Terraform changes land